src/lib/_common.scss

```diff
@@ -148,6 +140,12 @@ $header-height: 48px;
         flex: 1 1 auto;
         align-items: center;
         outline: none;
+
+        // Pseudo fullscreen for iOS Safari which doesn't support the fullscreen API
+        &.bp-fullscreen-unsupported {
+            position: fixed;
+            z-index: $overlay-z-index;
```

Comment:
set z-index to the highest z-index standard defined in BUIE which is 190 defined for overlays. This acts like an overlay when in full-screen mode